conda-recipe/build.sh

```diff
@@ -23,7 +23,7 @@ SKBUILD_ARGS=("${SKBUILD_ARGS[@]}" "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON")
 if [ "$CONDA_PY" == "36" ]; then
     WHEELS_BUILD_ARGS=("-p" "manylinux1_x86_64")
```

Comment:
I believe that needs to be removed, since env variable `CONDA_PY` is never used now, including any Jenkin job I and  because python 3.6 is out of support.

Reply:
Removed